src/App.js

```diff
@@ -4,9 +4,23 @@ import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export class App extends Component {
-  state = {};
+  state = {
+    query: '',
+  };
+
+  handleQueryChange = (event) => {
+    this.setState({
+      query: event.target.value,
+    });
+  }
 
   render() {
+    const { query } = this.state;
+    const normalizeQuery = query.toLowerCase();
+    const visibleMovies = moviesFromServer.filter(
+      movie => movie.title.toLowerCase().includes(normalizeQuery),
+    );
+
```

Comment:
```suggestion
    const normalizeQuery = query.toLowerCase();
    const visibleMovies = moviesFromServer.filter(
      movie => movie.title.toLowerCase().includes(normalizeQuery),
    );
```
Your code should filter any input register letters(IncEPtion === Inception).
Also, reread instructions. Filter by ``` title``` or ```description```

Reply:
Так він і фільтрує в любому регістрі )
фільтр по description добавив також)